src/main/java/net/pms/util/SubtitleUtils.java

```diff
@@ -622,4 +622,28 @@ public static String convertColourToASSColourString(int colour) {
 		outputString.append("&H").append(colourString.substring(6, 8)).append(colourString.substring(4, 6)).append(colourString.substring(2, 4));
 		return outputString.toString();
 	}
+
+	/**
+	 * Remove tags like <"b"> <"/b"> <"i"> <"/i"> <"u"> <"/u"> <"/font> <"font color="....">
+	 * from subtitles file for renderer not capable to show that tags correctly.
+	 *
+	 * @param file the source subtitles
+	 * @return Temporary file with the converted subtitles. This file will be deleted when the UMS will be stopped.
+	 */
+	public static File removeTagsFromSubs(File file) throws IOException {
+		BufferedReader input = FileUtil.bufferedReaderWithCorrectCharset(file);
+		File tempSubs = new File(configuration.getTempFolder(), file.getName());
```

Comment:
This is a potential bug - the same filename could already exist and be open for writing by another thread or by incorrect termination of UMS. This will result in an IOException and abort.